packages/babel-helpers/src/helpers.js

```diff
@@ -425,6 +425,51 @@ helpers.inheritsLoose = defineHelper(`
   }
 `);
 
+// Based on https://github.com/WebReflection/babel-plugin-transform-builtin-classes
+helpers.wrapNativeSuper = defineHelper(`
+  var _gPO = Object.getPrototypeOf || function _gPO(o) { return o.__proto__ };
+  var _sPO = Object.setPrototypeOf || function _sPO(o, p) { o.__proto__ = p };
```

Comment:
Bug! This fallback does not return `o`.

Reply:
Whoops, good catch!

Reply:
I just noticed that this hasn't been fixed in this PR, but it looks like it's fixed now.